packages/oui-numeric/src/index.spec.js

```diff
@@ -0,0 +1,155 @@
+
+    it('should increment value when clicking second button', () => {
+      let elt = angular.element('<oui-numeric model="foo"></oui-numeric>')
+      let scope = $rootScope.$new()
+      $compile(elt)(scope)
+      scope.foo = 10
+      scope.$digest()
+      angular.element(elt.find('button')[1]).triggerHandler('click')
+      expect(scope.foo).toBe(11)
+    })
+
+    it('should disable left button is value is lower or equal to min', () => {
+      let elt = angular.element('<oui-numeric model="foo" min="3"></oui-numeric>')
+      let scope = $rootScope.$new()
+      $compile(elt)(scope)
+      scope.foo = 5
+      scope.$digest()
+      expect(angular.element(elt.find('button')[0]).attr('disabled')).not.toBeDefined()
+      scope.foo = 3
+      scope.$digest()
+      expect(angular.element(elt.find('button')[0]).attr('disabled')).toBeDefined()
+      scope.foo = 1
+      scope.$digest()
+      expect(angular.element(elt.find('button')[0]).attr('disabled')).toBeDefined()
+    })
+
+    it('should disable right button is value is greater or equal to max', () => {
+      let elt = angular.element('<oui-numeric model="foo" max="5"></oui-numeric>')
+      let scope = $rootScope.$new()
+      $compile(elt)(scope)
+      scope.foo = 3
+      scope.$digest()
+      expect(angular.element(elt.find('button')[1]).attr('disabled')).not.toBeDefined()
+      scope.foo = 5
+      scope.$digest()
+      expect(angular.element(elt.find('button')[1]).attr('disabled')).toBeDefined()
+      scope.foo = 7
+      scope.$digest()
+      expect(angular.element(elt.find('button')[1]).attr('disabled')).toBeDefined()
+    })
+
+    it('should disable component when disabled attribut is set', () => {
+      let elt = angular.element('<oui-numeric model="foo" disabled="isDisabled"></oui-numeric>')
+      let scope = $rootScope.$new()
+      $compile(elt)(scope)
+      scope.foo = 3
+      scope.isDisabled = true
+      scope.$digest()
+      expect(angular.element(elt.find('input')[0]).attr('disabled')).toBeDefined()
+      expect(angular.element(elt.find('button')[0]).attr('disabled')).toBeDefined()
+      expect(angular.element(elt.find('button')[1]).attr('disabled')).toBeDefined()
+      scope.isDisabled = false
+      scope.$digest()
+      expect(angular.element(elt.find('input')[0]).attr('disabled')).not.toBeDefined()
+      expect(angular.element(elt.find('button')[0]).attr('disabled')).not.toBeDefined()
+      expect(angular.element(elt.find('button')[1]).attr('disabled')).not.toBeDefined()
+    })
+
+    it('should trigger onChange callback when value changes', () => {
+      let elt = angular.element('<oui-numeric model="foo" on-change="onChange($event)"></oui-numeric>')
+      let scope = $rootScope.$new()
+      let tmp
+      scope.foo = 5
+      scope.onChange = (ev) => {
+        tmp = ev.value
+      }
+      $compile(elt)(scope)
+      scope.$digest()
+      elt.find('input').controller('ngModel').$setViewValue('10')
+      expect(tmp).toBe(10)
```

Comment:
You can use `expect(scope.onChange).toHaveBeenCalledWith(10)` if you use `jasmine.createSpy`